.circleci/config.yml

```diff
@@ -954,6 +947,35 @@ jobs:
           path: ~/project/packages
 
 commands:
+  shallow-clone:
```

Comment:
is it necessary to have all these sub-step? can't it just be something like `git clone --depth 1 --branch <branch> ... ` in one go?

Reply:
Not if you want to checkout a specific commit shallowly